base/path.jl

```diff
@@ -196,6 +197,29 @@ function pathsep(paths::AbstractString...)
     return path_separator
 end
 
+"""
+    splitpath(path::AbstractString) -> (AbstractString, AbstractString, AbstractString...)
+
+Split a file path into all its path components. This is the opposite of
+`joinpath`. Returns a tuple of strings, one for each directory or file in the
+path, including the root directory if present.
+
+# Examples
+```jldoctest
+julia> splitpath("/home/myuser/example.jl")
+("/", "home", "myuser", "example.jl")
+```
+"""
+function splitpath(p::AbstractString)
+    out = ()
```

Comment:
This should return a `Vector{String}` not a tuple. That way you can also push stuff onto it.

Reply:
okay yeah, sounds good to me. i'll change that now! :)

Reply:
Done.